addons/cargo/CfgVehicles.hpp

```diff
@@ -426,4 +426,45 @@ class CfgVehicles {
             };
         };
    };
+   //Objects
+	class FloatingStructure_F; // External class reference
+    class RoadCone_F: thingX {
+        GVAR(size) = 1; // 1 = small, 2 = large
+        GVAR(canLoad) = 1;
+		XEH_ENABLED;
```

Comment:
What do you mean? All I want to say is that it's not indented correctly, you are using a tab here, it should be 4 spaces as per ACE3 standards.